src/format/parse.rs

```diff
@@ -564,28 +564,26 @@ mod tests {
     use crate::format::*;
     use crate::{DateTime, FixedOffset, TimeZone, Timelike, Utc};
 
-    #[test]
-    fn test_parse() {
-        use crate::format::InternalInternal::*;
-        use crate::format::Item::{Literal, Space};
-        use crate::format::Numeric::*;
+    fn parse_all(s: &str, items: &[Item]) -> ParseResult<Parsed> {
+        let mut parsed = Parsed::new();
+        parse(&mut parsed, s, items.iter())?;
+        Ok(parsed)
+    }
 
-        fn parse_all(s: &str, items: &[Item]) -> ParseResult<Parsed> {
-            let mut parsed = Parsed::new();
-            parse(&mut parsed, s, items.iter())?;
-            Ok(parsed)
-        }
+    macro_rules! check {
```

Comment:
Unrelated to this particular change, but since you're cleaning this area up: does this need to be a macro?

Reply:
You thought: let's ask something easy? :laughing:
I still use a macro to created the `Parsed` structs to compare against, but the test now uses more regular rust code.

Reply:
I didn't ask you to change anything about it. 😋